app/views/regions/v1/organisation.html

```diff
@@ -13,24 +13,20 @@
       <h1 class="nhsuk-heading-l">{{ organisation.name }}</h1>
 
       {% if organisation.status == 'Invited' %}
-        <p>You have invited this organisation to begin the process of using NHS Record a vaccination.</p>
-
-        <p>They will be able to set up users and vaccination sites.</p>
-
-        <p>They cannot start recording vaccinations until you activate them. You should do this once all relevant checks and contracts have been agreed.</p>
+        <p>You have invited this organisation to create an NHS Record a vaccination service account.</p>
 
       {% elseif organisation.status == 'Awaiting contract'  %}
         <p>This organisation has set up their users and is ready to use the service.</p>
 
-        <p>They cannot record vaccinations until you activate them. You should do this once all relevant checks and contracts have been agreed. </p>
+        <p>They cannot record vaccinations until you activate their account. You should do this once all relevant checks and contracts have been agreed. </p>
```

Comment:
Should we perhaps make it clear that you activate the whole organisation rather than the individual users? I wonder if it could be ambiguous?

```suggestion
        <p>They cannot record vaccinations until you activate the organisation’s account. You should do this once all relevant checks and contracts have been agreed. </p>
```